packages/oidc/jest.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   ...require("../../jest.config"),
+  testEnvironment: "jsdom",
```

Comment:
Might be worth leaving a comment that this package is only used by browser code so we only test in jsdom